### PR TITLE
Contact-form

### PR DESCRIPTION
**Pull request by AJ193**
### General requirements

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).

- Make sure that you used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).

- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

- HTML/CSS requirements

- Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

### Project requirements

- The section you need to build for this milestone is:
- the contact form:
- Form
- and the footer (if present in the design)
We included a screenshot of Template 1 for reference, but you should follow the template that you chose.

### Make sure that:

- Name field accepts up to 30 chars.

- Email field accepts only emails.

- Message field accepts up to 500 chars.

- All fields have placeholder text.

- All fields have the correct type.

- A user cannot submit the form if one or more of the fields is empty.

- Collect data by using the [Formspree service](https://formspree.io/html).

- In order to lay out the elements on the page you should use Flexbox.

- You don't need to implement any functionality that requires JavaScript.

- You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -283,6 +283,40 @@ <h4 class="about-me">About me</h4>
...
+        </div>
+        <div class="message-text">
+            <textarea class="border-none-3" name="message" rows="8" cols="39" maxlength="500" placeholder="Enter text here" required></textarea>
+        </div>
+
+        <button class="form-button" type="submit" button-interaction="hover">         
+            <h3 class="form-button-text">Get in touch</h3>
+        </button>
+    </form>
+</section>
```
Comment (ABDULALI3468) [inline index.html:308]: Please consider using the same styles as used in the Figma design to make your project requirement increase user experience and make it professional.

## Figma Design:-

image


## Your Design:-

image

File: index.html [hunk 2]
```diff
@@ -283,6 +283,40 @@ <h4 class="about-me">About me</h4>
...
+            <input class="border-none-2" type="email" name="email" placeholder="Email address" required>
+            <hr class="container-2">
+        </div>
+        <div class="message-text">
+            <textarea class="border-none-3" name="message" rows="8" cols="39" maxlength="500" placeholder="Enter text here" required></textarea>
+        </div>
+
+        <button class="form-button" type="submit" button-interaction="hover">         
+            <h3 class="form-button-text">Get in touch</h3>
+        </button>
```
Comment (ABDULALI3468) [inline index.html:306]: Please consider adding the correct hover and click functionality on the button to make it closer to the design. Remember the goal of the project is to make a clone of the design. Using your own sense for styles and others is not part of this project.

## Hover(:hover) and Press(:active) styles:-

image

## Conclusion:- 

When the button is hovered or pressed it needs to be exactly in style as the above diagram illustrates.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @AJ193 , 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

**1-** Please consider resolving the horizontal scroll on your website to increase user experience and professionalism.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @AJ193, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- ✅Project is professionally documented.
- ✅The right GitHub flow method has been used.
- ✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**